app/main.py

```diff
@@ -14,24 +14,34 @@ def load_settings():
 
 
 settings = load_settings()
-
-if __name__ == "__main__":
-    # Only for debugging while developing
-    app.run(host='0.0.0.0', debug=True, port=5000)
+yt = YouTrackHelper(settings["youtrack_instance_name"], settings["youtrack_token"])
 
 
 @app.route('/pull-request/', methods=['POST'])
 def assign():
+    users = settings["users_dict"]
     payload = request.get_json()
     pr = payload["pull_request"]
-    assignee = pr["assignee"]
-
-    if payload["action"] not in ["assigned", "opened"] or assignee is None:
-        return '', 200
-
     issue_id = pr["head"]["ref"]
```

Comment:
Do you think it's worth doing the following bits of cleaning here:

* strip off everything after an underscore (because we have used `iXXXX_foo` with metadata in the past and that's been useful sometimes)
* replace an initial `i` with `VIMC-` so that `i123` becomes `VIMC-123`.  This would treat vimc tickets specially but might be less disruptive

I don't feel super strongly about this but it feels like given we don't know how well this would work it might be nice to separate adding the new monitoring in from the change of workflow

Reply:
Both good ideas, yes. Also means we won't yet have to update the release tool to recognise full ids instead of `ixxx`.